jazz-ui/Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+pipeline {
+  agent any
+
+  environment {
+      creds = credentials('jenkins1cred')
+  }
+	stages {
+		stage('Checkout Jazz-UI') {
+			steps {
+				script {
+					sh "if [ -d jazz-ui ] ; then rm -rf jazz-ui; fi"
+					def var_bitbucket_framework_url = "http://${creds_USR}:${creds_PSW}" + "@" + env.REPO_BASE + "/scm/" + env.REPO_CORE
```

Comment:
we are trying to **not** name our variables with var_ going forward. Also call it scm_base_url as we now have gitlabs coming in.